StoreApp.DAL/EF/StoreContext.cs

```diff
@@ -27,8 +27,8 @@ public class StoreDbInitializer : CreateDatabaseIfNotExists<StoreContext>
     {
         protected override void Seed(StoreContext db)
         {
-            db.Products.Add(new Product { Name = "TestProduct",  Quantity = 1 });
-            db.SaveChanges();
+            //db.Products.Add(new Product { Name = "TestProduct",  Quantity = 1 });
+            //db.SaveChanges();
```

Comment:
Are you sure that you need to keep this code?

Reply:
fixed